content/html/concepts/tags/terms/q/q.md

```diff
@@ -0,0 +1,39 @@
+---
+Title: '<q>'
+Description: 'Used to represent a brief inline quotation.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Tags'
+  - 'Attributes'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<q>` tag is used to represent a brief inline quotation, as opposed the [`<blockquote>` tag](https://www.codecademy.com/resources/docs/html/tags/blockquote) being used for longer ones.
+
+## Syntax
+
+```html
+<q cite=""><!-- brief quotation goes inside the tag --></q>
```

Comment:
For these comments, I'm thinking that we should capitalize the first word and end with a punctuation~

Reply:
```suggestion
<q cite=""><!-- Brief quotation goes inside the tag. --></q>
```